src/libsyntax/attr.rs

```diff
@@ -77,6 +81,93 @@ pub fn is_used(attr: &Attribute) -> bool {
     })
 }
 
+pub trait AttrMetaListItemMethods {
```

Comment:
`ListMetaMethods` should probably return the literal as a string for `name()` instead of a `None` (cc @nrc)